manifests/integrations/tcp_check.pp

```diff
@@ -0,0 +1,129 @@
+# Class: datadog_agent::integrations::tcp_check
+#
+# This class will install the necessary config to hook the tcp_check in the agent
+#
+# Parameters:
+#   name
+#       (Required) - Name of the service. 
+#        This will be included as a tag: instance:<name>.
+#
+#   host
+#       (Required) - Host to be checked. 
+#        This will be included as a tag: url:<host>:<port>.
+#
+#   port
+#       (Required) - Port to be checked. 
+#        This will be included as a tag: url:<host>:<port>.
+#
+#   timeout
+#       (Optional) - Timeout for the check. Defaults to 10 seconds.
+#
+#   threshold
+#       (Optional) - Used in conjunction with window. An alert will 
+#        trigger if the check fails <threshold> times in <window> attempts.
+#
+#   window
+#       (Optional) - Refer to �threshold�.
```

Comment:
@aepod the quotes here are not ASCII quotes, this is what's preventing the CI from running your tests. You should probably change them to regular quotes, or just remove them.